### DISCUSSION TITLE
LoTW upload work but qsos not uploaded

### DISCUSSION THREAD

**Discussion by fg8oj** [magicbug/Cloudlog#1201] [General]
Hello,
Just installed an instance of Cloudlog for a friend and saw the great work of Peter.
I didn't solved a problem, I can manually or automatically use the upload lotw page and get results of confirmed qsos but it doesn't upload new qso on logbook. I really don't know how upload work (I don't have tqsl installed and I don't know if it needed).
Thanks for you help.
Burt FG8OJ

**Comment by dg9vh** [comment]
Hi,

LoTW only works on manual upload/download or per cronjob: https://github.com/magicbug/Cloudlog/wiki/Recommended-Cron-Jobs

It does not upload QSO at the moment when logging them.

**Comment by fg8oj** [comment]
Many thanks Kim for this quick answer.
So crontab and manual are working fine !
I will add a wsjtx to cloudlog script to sync both and let my friend do manual tqsl uploads.
73, Burt FG8OJ

**Comment by magicbug** [comment]
LoTW upload isn't real-time, you need to call the cronjob I recommend setting it up as per the setup guide which is every hour, that uploads QSOs and downloads any matches.

**Comment by fg8oj** [comment]
Hi Peter, 
cronjob have been, thank you!
73, Burt FG8OJ

**Reply by magicbug** [reply]
Reason for it not being real-time was the added "page load time" being as PHP doesn't do background jobs so easily out the box, thought it was best to upload hourly or so :)
